src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/EnumExtensions.cs

```diff
@@ -8,7 +8,7 @@ public static class EnumExtensions
     {
         public static string GetEnumLabel(this Enum enumValue)
         {
-            return GetEnumLabelValueAttribute(enumValue)?.Label ?? enumValue.ToString();
+            return GetEnumLabelAttribute(enumValue)?.Label ?? enumValue.ToString();
         }
```

Comment:
See my comment below about `GetEnumLabelAttribute` being identical to `GetEnumLabelValueAttribute`

Reply:
Cheers - will fix

Reply:
Done